DEVELOPMENT.md

```diff
@@ -5,65 +5,76 @@ Errors are done with `thiserror`. We adopt a minimalistic approach to errors.
 * We try to have _one error kind_ for the entirety of Loco.
 * Errors that cannot be handled, are _informative_ and so can be opaque (we don't offer deep matching on those)
 * Errors that can be handled and reasoned upon should be able to be matched and extract good knowledge from
-* To users, error should _not be cryptic_, and should indicate how to fix issues as much as possible, or point to the issue precisely
-
+* To users, error should _not be cryptic_, and should indicate how to fix issues as much as possible, or point to the
+  issue precisely
 
 ### Auto conversions
 
 When possible use `from` conversions.
 
 ```rust
-    #[error(transparent)]
-    JSON(#[from] serde_json::Error),
+    # [error(transparent)]
+JSON( # [from] serde_json::Error),
 ```
 
-When complicated, implement a `From` trait yourself. This is done to _centralize_ errors into one place and not litter needless `map_err` code which holds error conversion logic (an exception is Context, see below).
-
+When complicated, implement a `From` trait yourself. This is done to _centralize_ errors into one place and not litter
+needless `map_err` code which holds error conversion logic (an exception is Context, see below).
 
 ### Context
 
-When you know a user might need context, resort to manually shaping the error with extra information. First, define the error:
+When you know a user might need context, resort to manually shaping the error with extra information. First, define the
+error:
 
 ```rust
-    #[error("cannot parse `{1}`: {0}")]
-    YAMLFile(#[source] serde_yaml::Error, String),
+    # [error("cannot parse `{1}`: {0}")]
```

Comment:
something strange with formatting here?
we have a space after `#`, `&`, etc.

Reply:
Fixed.